pages/wg/dei.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: page
+title: Diversity, Equity, & Inclusion Working Group
+subtitle:
+permalink: /wg/dei/
+---
+
+# Overview
+
+The US-RSE Association is committed to providing an inclusive environment with equitable treatment for all and to promoting and encouraging diversity throughout the RSE community in the US. The US-RSE Diversity, Equity, and Inclusion (DEI) working group is for those who are interested in regular discussions regarding DEI in the RSE community. The goal of the DEI working group is to promote the values of the US-RSE DEI Mission Statement with a particular focus on promoting opportunities for DEI education to the wider community. The group meets bi-weekly via Zoom and welcomes all newcomers.
+
+To get involved with the DEI working group, visit the [`#dei-discussion`](https://usrse.slack.com/messages/dei-discussion) channel on the US-RSE slack, or contact the <a href="mailto:dei_wg@us-rse.org">DEI working group list</a>.
+
+
+# Work Products
+
+The US-RSE DEI working group has produced and hosts the following documents and events.
+
+## DEI Mission Statement
+
+The US-RSE DEI working group's first large initiative was to create a US-RSE DEI Mission Statement to integrate into the US-RSE core values. This DEI Mission Statement was published in August 2021 on the [DEI Mission Statement page]({{ site.baseurl }}/about/dei-mission). We expect this mission statement to evolve and mature as the US-RSE community grows and welcome feedback.
+
+## DEI Speaker Series
+
+The US-RSE DEI working group invites DEI speakers to present on DEI topics. In 2021, we hosted [Dr. Cherri Pancake]({{ site.url }}/events/2021/2021-03-dei-speaker-series/) as she presented her talk, "Can We Change the Face of Computing?" and [Joanne Esch and Andrea Bowen-Jones]({{ site.url }}/events/2021/2021-07-dei-speaker-series-ncwit) of the National Center for Women & Information Technology.
+
+We will continue to invite and host speakers as they are available. We welcome suggestions regarding potential speakers.
+
+## DEI Media Club
+
+The US-RSE DEI working group hosts monthly DEI Media Club discussions in which we invite participants to read, listen to, or watch the chosen monthly DEI media and come ready to discuss the content as a small group.
+
```

Comment:
Do we want to add details here about prevents events, like for the speaker series?

E.g.,
```
Media that the club discussed in 2021 included the book [Better Allies](links here) as organized by [Nicole Brewer]() and podcasts [x]() and [y]() organized by [Miranda Mundt]()
```